src/dependencies.scala

```diff
@@ -150,5 +173,52 @@ object Dependencies {
         deps map { x => x: ClasspathDep[ProjectReference] }:_*)
     }
   }
+
+  implicit class ModuleIDOps(id: ModuleID)
+  {
+    def revMatches(other: String) = {
+      def partMatches(p: (String, String)) = p._1 == p._2 || p._1 == "+"
+      val parts = id.revision.toString.split('.')
+      val otherParts = other.toString.split('.')
+      val partsMatch = parts zip(otherParts) forall(partMatches)
+      partsMatch && (
+        parts.length == otherParts.length ||
+        (parts.length < otherParts.length) && parts.lastOption.exists(_ == "+")
```

Comment:
I'm not thinking this through, but should vice versa also be considered for `otherParts.length < parts.length && otherParts.lastOption.exists ...`?